mqtt/build/linux/buildImage.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# This script copies the iodedged executable files that go into the azureiotedge-iotedged image,
+# for each supported arch (x86_64, arm32v7, arm64v8).
+# It then publishes executable files along with their corresponding dockerfiles to the publish directory,
+# so that buildImage.sh can build the container image.
+
+set -e
+###############################################################################
+# Define Environment Variables
+###############################################################################
+
+ARCH=$1
+DOCKER_REGISTRY=$2
+DOCKER_USERNAME=$3
+DOCKER_PASSWORD=$4
+DOCKER_IMAGEVERSION=${BUILD_BUILDNUMBER:=$5}
+
+if [[ "$ARCH" == "amd64" ]]; then
```

Comment:
Please make sure scripts you added has similar structure, arguments and values similar across all projects https://github.com/Azure/iotedge/blob/master/scripts/linux/buildEdgelet.sh#L31-L43